pyscf/sftda/numint2c_sftd.py

```diff
@@ -0,0 +1,86 @@
+#/usr/bin/env python
+# Copyright 2014-2024 The PySCF Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file can be merged into pyscf.dft.numint2c.py
+
+MGGA_DENSITY_LAPL = False # just copy from pyscf.dft.numint2c.py
+
+import functools
+import numpy as np
+from pyscf import lib
+from pyscf.dft import numint
+
+from pyscf.dft.numint2c import __mcfun_fn_eval_xc
```

Comment:
Copy the code of `__mcfun_fn_eval_xc` to this module. This private function might be deleted or changed in the future.

Reply:
Thanks. I will change that in the new commit.